crates/ruma-events-macros/src/event_enum.rs

```diff
@@ -1071,3 +1071,40 @@ impl EventEnumEntry {
         Ok(EventEnumVariant { attrs, ident })
     }
 }
+
+pub(crate) fn expand_from_impls_derived(input: DeriveInput) -> TokenStream {
+    let variants = match &input.data {
+        Data::Enum(DataEnum { enum_token: _, brace_token: _, variants }) => variants,
```

Comment:
No need to mention unused fields explicitly, this can just be `{ variants, .. }`.